.github/workflows/v-next-ci.yml

```diff
@@ -6,18 +6,35 @@ on:
     branches:
       - v-next
   pull_request:
-    # TODO: Remove the filter when v-next is merged into main
-    branches:
-      - v-next
```

Comment:
We could add a `paths` filter here. It wouldn't be perfect but running CI whenever `v-next` folder is modified should catch the vast majority of the changes.

```
    paths:
      - .github/workflows/v-next-ci.yml
      - v-next/**
```